From 156e9ef5a5072a5f623826a71d749b72b1e5c62e Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 12 Dec 2024 14:09:34 +0800
Subject: [PATCH 322/347] AOSCOS: MIPS: mark CPU_LOONGSON64 as HAVE_PLAT_MEMCPY

loongson3-memcpy is a platform-specific memcpy implementation, add
HAVE_PLAT_MEMCPY under CPU_LOONGSON64.

Fixes: "AOSCOS: memcpy optimization for loongson-3"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index dae56c154434..5c41c5950bc2 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -1369,6 +1369,7 @@ config CPU_LOONGSON64
 	select MIPS_FP_SUPPORT
 	select GPIOLIB
 	select SWIOTLB
+	select HAVE_PLAT_MEMCPY
 	help
 	  The Loongson GSx64(GS264/GS464/GS464E/GS464V) series of processor
 	  cores implements the MIPS64R2 instruction set with many extensions,
-- 
2.51.0

